extensions/default/src/findViewportsByPosition.ts

```diff
@@ -17,7 +17,7 @@ import { StateSyncService, Types } from '@ohif/core';
 export const findOrCreateViewport = (
   hangingProtocolService,
   viewportsByPosition,
-  viewportIndex: number,
+  viewportId: string,
```

Comment:
Looking at ViewportGridProvider, I believe this argument is `pos` (i.e. position) and not `viewportId`, no?

Reply:
that is not what we use for `findViewportsByPositoin` it is getting bound as layoutFindOrCreate

Reply:
I am fairly certain that ViewportGridProvided does call this `findOrCreateViewport` function, but notice that the `viewportId` argument is NOT actually used, but if it were it would instead be `position`. @wayfarer3130 could you please confirm this?